core/imt/test/CMakeLists.txt

```diff
@@ -5,4 +5,5 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.
 
 ROOT_ADD_GTEST(testTaskArena testRTaskArena.cxx LIBRARIES Imt ${TBB_LIBRARIES} FAILREGEX "")
-ROOT_ADD_GTEST(testTBBGlobalControl testTBBGlobalControl.cxx LIBRARIES Imt ${TBB_LIBRARIES})
+ROOT_ADD_GTEST(testTBBGlobalControl testTBBGlobalControl.cxx testEnableImt.cxx LIBRARIES Imt ${TBB_LIBRARIES})
+ROOT_ADD_GTEST(testEnable testEnableImt.cxx LIBRARIES Imt ${TBB_LIBRARIES})
```

Comment:
Why is `testEnableImt.cxx` compiled into two tests? Can we get a more descriptive name than `testEnable`, at least `testEnableImt`?